MobileFilter/sections/specific_web.txt

```diff
@@ -437,7 +437,7 @@ m.megafonpro.ru##div[class="v y m"]
 mamibai.com##.content-ad-spa
 mobil.ksta.de##.is-ad
 sueddeutsche.de##power-stories-embed
-mbasic.facebook.com##div[role="article"][data-ft*='"originated_from_recommendation":"1"']
+mbasic.facebook.com,d.facebook.com,free.facebook.com,free.facebookcorewwwi.onion,mbasic.facebookcorewwwi.onion,d.facebookcorewwwi.onion,m.facebook.com,m.facebookcorewwwi.onion,b-m.facebook.com,b-m.facebookcorewwwi.onion,mobile.facebook.com,mobile.facebookcorewwwi.onion##div[role="article"][data-ft*='"originated_from_recommendation":"1"']
```

Comment:
free.facebook.com gets redirected to m.facebook.com, so rules with free. not required imho.

Reply:
@BlazDT on my end I have the same.

Reply:
`free.facebook.com` and `0.facebook.com` is apparently only properly accessible in some parts of east Asia.

However, using Tor Browser and visiting `free.facebookcorewwwi.onion` and `0.facebookcorewwwi.onion`, will most likely prove that the `free.` and `0.` prefixes are actually in use by `Facebook.com